Lib/fontTools/svgLib/path/parser.py

```diff
@@ -317,17 +317,15 @@ def parse_path(pathdef, pen, current_pos=(0, 0)):
                 end += current_pos
 
             svg_arc = Arc(current_pos, radius, rotation, arc, sweep, end)
-            arc_points = []
+            arc_points = [(current_pos.real, current_pos.imag)]
```

Comment:
actually, you don't have to pass the current_pos to qCurveTo. That's implied by the preceding moveTo. If you do that, you end up with a duplicate point (see my other comment)